appendices/rdp-controller.md

```diff
@@ -0,0 +1,35 @@
+# RDP Controller
+
+RDP stands for Remote Desktop Protocol and is the foundation for remote desktop connections in Windows. With RDP you can connect to a remote computer running Windows and use it as if it was your local machine.
+
+Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.
+
+In order to start using the RDP Controller, you have to follow these steps:
+
+1. Download and run [this file]() on your local Windows system, answering *Yes* to the security warning message.
```

Comment:
`[this file]()` 
there's no path to this file

Reply:
don't you say :smile:
look at my first comment

Reply:
```suggestion
1. Download and run [this file](../-assets/RDP local machine.reg) on your local Windows system, answering *Yes* to the security warning message.
```